src/mpid/ch3/src/mpidi_pg.c

```diff
@@ -980,12 +976,6 @@ int MPIDI_PG_Close_VCs( void )
                 MPIR_ERR_CHECK(mpi_errno);
 	    } else if (vc->state == MPIDI_VC_STATE_INACTIVE ||
                        vc->state == MPIDI_VC_STATE_MORIBUND) {
-                /* XXX DJG FIXME-MT should we be checking this? */
-                if (MPIR_Object_get_ref(vc) != 0) {
-		    /* FIXME: If the reference count for the vc is not 0,
-		       something is wrong */
-                    MPIDI_PG_release_ref(pg, &inuse);
-                }
```

Comment:
Shouldn't we replace them with an assertion `MPIR_Assert(MPIR_Object_get_ref(vc) == 0);`?

Reply:
Will do.

Reply:
Did you add the assertions?